lightning/src/ln/outbound_payment.rs

```diff
@@ -1789,6 +1772,54 @@ impl OutboundPayments {
 		}
 	}
 
+	fn mark_invoice_received(
+		&self, payment_id: PaymentId, payment_hash: PaymentHash, with_manual_handling: bool
+	) -> Result<(Retry, RouteParametersConfig), Bolt12PaymentError> {
+		let retry_strategy;
+		let params_config;
```

Comment:
It would be cleaner to simply return directly instead of needing to assign to these first.

Reply:
Done! [pr3658.03](https://github.com/shaavan/rust-lightning/commits/pr3658.03)
Thanks for the pointer!